server/routes/handlers/outcomes/getCasesInProgressHandler.js

```diff
@@ -7,19 +9,25 @@ const getCasesInProgressHandler = caseService => async (req, res) => {
     session
   } = req
 
-  const filters = getOutcomeTypesListFilters(req.query)
-  const filtersApplied = filters.map(filterObj => filterObj.items.filter(item => item.checked).length).pop()
+  const response = await caseService.getOutcomesList(courtCode, req.query, sorts, state)
```

Comment:
`req.query` is used a couple of times - should we make it a variable on line 6?

Reply:
done